x/dex/keeper/core.go

```diff
@@ -299,6 +300,8 @@ func (k Keeper) MultiHopSwapCore(
 		bestRoute.route,
 	))
 
+	gasConsumed(ctx)
```

Comment:
Not sure what this is tracking. 
In the initial spec I had talked about tracking gas per swap which would measure gas used by calls into `k.Swap`

Reply:
https://www.notion.so/hadron/Duality-Monitoring-a9c194e4e81f490dbf97ca39fee7ad2e?pvs=4#bcdc3d47c3a74c9f9fe2a0d994ec4406